package.json

```diff
@@ -12,8 +12,9 @@
     "image:build": "./scripts/build-image.sh",
     "image:push": "./scripts/push-image.sh",
     "clean": "yarn rimraf dist",
-    "lint": "yarn eslint . --ext .js,.jsx,.ts,.tsx",
-    "lint:fix": "yarn eslint . --ext .js,.jsx,.ts,.tsx --fix ",
+    "lint": "yarn eslint . --ext .js,.jsx,.ts,.tsx && yarn prettier ./src/**/*.scss ./src/**/*.json --check",
+    "lint:fix": "yarn eslint . --ext .js,.jsx,.ts,.tsx --fix",
+    "prettier:fix": "yarn prettier ./src/**/*.scss ./src/**/*.json --write",
```

Comment:
As mentioned on Slack, let's move this `yarn prettier [...] --write` command to be part of `lint:fix` and get rid of `prettier:fix`. If we do that and delete the unused `tslint.json` file I'll approve the PR

Reply:
(and optionally we could have a `prettier:fix` that instead runs `yarn prettier . --write`. Although that would include .yaml files, which maybe we should add above?)

Reply:
@mturley, that makes sense, I've added a commit to the patch, thanks.